src/scope.litcoffee

```diff
@@ -16,7 +16,6 @@ that should be output as part of variable declarations.
 
       constructor: (@parent, @expressions, @method, @referencedVars) ->
         @variables = [{name: 'arguments', type: 'arguments'}]
-        @comments  = {}
```

Comment:
Not that this is wrong, but why is this removed? How are we tracking comments now?